### PR TITLE
examples: Convert color_patches, conways_game_of_life, epstein_civil_violence, forest_fire to simple namespace

### PR DESCRIPTION
**Pull request by rht**

### CODE REVIEW

File: examples/forest_fire/forest_fire/server.py [hunk 1]
```diff
@@ -18,19 +16,23 @@ def forest_fire_portrayal(tree):
...
-pie_chart = PieChartModule(
+pie_chart = mesa.visualization.PieChartModule(
     [{"Label": label, "Color": color} for (label, color) in COLORS.items()]
 )
 
 model_params = {
     "height": 100,
     "width": 100,
-    "density": UserSettableParameter("slider", "Tree density", 0.65, 0.01, 1.0, 0.01),
+    "density": mesa.visualization.UserSettableParameter(
```
Comment (tpike3) [inline examples/forest_fire/forest_fire/server.py:32]: Appreciating it was my delay... did you want to update this to `mesa.visualization.Slider`

Comment (rht) [reply examples/forest_fire/forest_fire/server.py:32]: I will do that in a separate PR.

**Comment by tpike3** [comment]
LGTM

**Review by tpike3 — COMMENTED** [review]
LGTM

**Pull request merged**